src/vs/workbench/services/extensions/common/extensionsRegistry.ts

```diff
@@ -652,7 +652,7 @@ schemaRegistry.registerSchema(productSchemaId, {
 					items: {
 						type: 'string',
 						enum: Object.keys(allApiProposals),
-						markdownEnumDescriptions: Object.values(allApiProposals)
+						markdownEnumDescriptions: Object.values(allApiProposals).map(value => value.proposal)
```

Comment:
We should also add the latest version of the proposal to the API, e.g have `fooProposal` and `fooProposal@3`. This ensures extensions with `fooProposal@1` get a squiggle

Reply:
This is in `product.json` where we do not want to add versions. I added `version` to the `package.json` schema here already - 

https://github.com/microsoft/vscode/blob/e3e51b079f52f4262a9d3698ccf1ad8ba14335a4/src/vs/workbench/services/extensions/common/extensionsRegistry.ts#L245